### PR TITLE
Update deliverability.md

### PR DESCRIPTION
**Pull request by tanhengyeow**
**Description of the change**:
Improve spelling errors, formatting and grammar mistakes.

### CODE REVIEW

File: content/docs/ui/sending-email/deliverability.md [hunk 1]
```diff
@@ -73,10 +73,10 @@ With email, things don't happen overnight, and magic wands are few and far betwe
 
 Encouraging your recipients to do certain things can help bolster the trust ISPs have for you and your messages. Some examples can include:
 
-- "Add us to your address book!" - Having a recipient add your from address you their address book or trusted senders list can go a long way. More often than not, if one of an ISPs recipients trust a sender, they will be more lenient to similar messages to different recipients!
+- "Add us to your address book!" - Having a recipient add you from their address book or trusted senders list can go a long way. More often than not, if one of an ISPs recipients trust a sender, they will be more lenient to similar messages to different recipients!
```
Comment (Whatthefoxsays) [inline content/docs/ui/sending-email/deliverability.md]: Should read "Having a recipient add your *from* address to their address book or trusted senders list can go a long way."

File: content/docs/ui/sending-email/deliverability.md [hunk 2]
```diff
@@ -73,10 +73,10 @@ With email, things don't happen overnight, and magic wands are few and far betwe
...
-- "If you don't receive an email right away, please check your spam folder and mark "not spam"" - Adding this simple sentence to your sign-up form area can solve a lot of potential heartache. If a message you sent ends up in the Spam folder, and the recipient manually goes in and pulls it out, that's fantastic! This not only helps an ISPs incoming mail filters in avoiding false positives, but also improves your standing with that ISP.
+- "If you don't receive an email right away, please check your spam folder and mark "not spam" - Adding this simple sentence to your sign-up form area can solve a lot of potential heartaches. If a message you sent ends up in the Spam folder, and the recipient manually goes in and pulls it out, that's fantastic! This not only helps an ISPs incoming mail filters in avoiding false positives, but also improves your standing with that ISP.
```
Comment (Whatthefoxsays) [inline content/docs/ui/sending-email/deliverability.md]: heartache should remain singular.

File: content/docs/ui/sending-email/deliverability.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Think of it this way: would you rather a recipient politely decline future email
 
  ### 	Who are you?
 
-Maximum company visibility helps as well. Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and sender of email. This also helps your recipients know that this message is indeed from you!
+Maximum company visibility helps as well. Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and sender of an email. This also helps your recipients know that this message is indeed from you!
```
Comment (Whatthefoxsays) [inline content/docs/ui/sending-email/deliverability.md]: To avoid confusion, I would rephrase the second sentence to:

"Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and email sender."

**Comment by thinkingserious** [comment]
Hello @tanhengyeow,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by tanhengyeow** [comment]
@ksigler7 I completed 4 `difficulty: easy` PRs (not including this one) and this would potentially be my 5th `difficulty: easy` PR made. 

Given that the difficulty is still relatively easy, can this be marked as `difficulty: easy` instead? Or does it not matter to qualify for Sendgrid's Hacktoberfest swag? Do let me know, thank you 😄
